cbor/shared/src/test/scala/fs2/data/cbor/high/ValueParserTest.scala

```diff
@@ -0,0 +1,16 @@
+package fs2.data.cbor.high
+
+import cats.effect.IO
+import fs2._
+import weaver.SimpleIOSuite
+
+object ValueParserTest extends SimpleIOSuite {
```

Comment:
I started some Scalacheck-based roundtrip testing in http4s-fs2-data, but maybe it's better done here. I'll port it over or at least extend this test to capture positive bigints as well.